react/Coupon.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState }  from 'react'
+import { Button, Input, Tag } from 'vtex.styleguide'
+import { compose, graphql } from 'react-apollo'
+
+import * as Coupons from './graphql/coupons.graphql'
+
+import styles from './styles.css'
+
+const Coupon: StorefrontFunctionComponent<CouponProps> = ({ }) => {
+
+  const [isShowingPromoButton, setIsShowingPromoButton ] = useState(true);
```

Comment:
Check if your IDE linting is enabled.
```suggestion
  const [isShowingPromoButton, setIsShowingPromoButton] = useState(true);
```